eos/utils/concrete-cacheable-observable.hh

```diff
@@ -311,6 +312,15 @@ namespace eos
 
             virtual ObservablePtr make(const Parameters & parameters, const Kinematics & kinematics, const Options & options) const
             {
+                for (const auto & fo : _forced_options)
+                {
+                    const auto & key = std::get<0>(fo);
+                    if (options.has(key))
+                    {
+                        Log::instance()->message("[ConcreteObservableEntry.make]", ll_error)
```

Comment:
Shouldn't this rather be `[ConcreteCacheableObservableEntry.make]`?

Reply:
Indeed.